Jenkinsfile.ad-hoc

```diff
@@ -0,0 +1,348 @@
+    }
+    id = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+    baseImageName ="browser-f/android:${dockerTag}"
+    codeImageName ="browser-f/android:code"
+    baseImage = stage('Build base image') {
+        docker.build(baseImageName, '--build-arg UID=`id -u` --build-arg GID=`id -g` -f Dockerfile .')
+    }
+    stage('Build code image') {
+        writeFile file: 'Dockerfile.code', text: """
+            FROM ${baseImageName}
+            COPY --chown=jenkins:jenkins . /app/
+        """
+
+        codeImage = docker.build(
+            codeImageName,
+            '-f Dockerfile.code .',
+        )
+    }
+}
+
+def build(Map m){
+    def flavorname = m.name
+    def buildarch = m.arch
+    def brandname = m.brand
+    def cliqzchannel = m.channel
+    def androidtarget = m.target
+    def getImageName = m.getImageName
+    def buildParam = m.buildParam
+    def nodeLabel = 'master'
+    def getId = m.getId
+    def apk = ""
+    def gradleflavor = ""
+    if (brandname=="bond"){
+        gradleflavor = "Bond"
+    } else {
+        gradleflavor = "Photon"
+    }
+    return {
+        if (buildParam) {
+            node(nodeLabel) {
+                getImageName().inside() {
+                    stage('Download cache') {
+                            withCredentials([
+                                [
+                                    $class: 'UsernamePasswordMultiBinding',
+                                    credentialsId: 'f1732e5f-3e84-47ad-9286-a5fa6657ec43',
+                                    passwordVariable: 'AWS_SECRET_ACCESS_KEY',
+                                    usernameVariable: 'AWS_ACCESS_KEY_ID',
+                                ]
+                            ]) {
+                                def s3Path = "s3://repository.cliqz.com/dist/android/cache"
+                                def cachePath = ".gradle/caches"
+                                sh """#!/bin/bash -l
+                                    set -e
+                                    set -x
+                                    cd
+                                    aws s3 sync --acl public-read --acl bucket-owner-full-control ${s3Path} ${cachePath}
+                                """
+                            }
+                        }
+                    stage("Build APK: ${flavorname}") {
+                        withEnv([
+                            "ANDROID_TARGET=${androidtarget}",
+                            "BRAND=${brandname}",
+                            "CLIQZ_CHANNEL=${cliqzchannel}"
+                        ]){
+                            sh '''#!/bin/bash -l
+                                set -e
+                                set -x
+                                cd /app/mozilla-release
+                                cp ../mozconfigs/deploy.mozconfig mozconfig
+                                export HOST_CC=$CLANG_HOME/bin/clang
```

Comment:
Please, remove this. It is not needed anymore.

Reply:
Done.